content/hi/mutual-transport-layer-security.md

```diff
@@ -0,0 +1,25 @@
+---
+title: एमटीएलएस (Mutual Transport Layer Security)
+status: Completed
+category: अवधारणा
+tags: ["security", "", ""]
+---
+
+## यह क्या है
+
+म्युचुअल टीएलएस (एमटीएलएस) एक तकनीक है जिसका उपयोग दो [सेवाओं](/service/) के बीच भेजे गए संदेशों को प्रमाणित और एन्क्रिप्ट करने के लिए किया जाता है।
+म्यूचुअल टीएलएस मानकी [ट्रांसपोर्ट लेयर सिक्योरिटी](/transport-layer-security/) (टीएलएस) प्रोटोकॉल है लेकिन,
+केवल एक कनेक्शन की पहचान को मान्य करने के बजाय, दोनों पक्षों को मान्य किया जाता है।
+
+## समस्या
+
+[माइक्रोसर्विसेज](/microservices/) एक नेटवर्क पर संचार करता हैं और,
+आपके वाईफाई नेटवर्क की तरह, उस नेटवर्क पर संचार पारगमन को हैक किया जा सकता है।
+एमटीएलएस यह सुनिश्चित करता है कि कोई भी अनधिकृत पार्टी वैध अनुरोधों को नहीं सुन सकते है या उनका प्रतिरूपण नहीं कर सकते है।
+
+## समाधान
+
+mTLS सुनिश्चित करता है कि क्लाइंट और सर्वर के बीच ट्रैफिक दोनों दिशाओं में सुरक्षित और विश्वसनीय है,
+नेटवर्क या एप्लिकेशन में लॉग इन करने वाले उपयोगकर्ताओं के लिए सुरक्षा की एक अतिरिक्त परत प्रदान करता है।
+यह उन क्लाइंट डिवाइस के साथ कनेक्शन को भी सत्यापित करता है जो लॉगिन प्रक्रिया का पालन नहीं करता हैं, जैसे इंटरनेट ऑफ थिंग्स (IoT) डिवाइस।
+ऑन-पाथ अटैक, स्पूफिंग अटैक, क्रेडेंशियल स्टफिंग, ब्रूट फोर्स अटैक इत्यादि जैसे हमलों को एमटीएलएस द्वारा रोका जा सकता है।
```

Comment:
These sentences can be made to appear in continuation without the extra new line.

Reply:
device is called `उपकरण`